server/kit/sd_log.go

```diff
@@ -27,34 +28,51 @@ func isGAE() bool {
 	return os.Getenv("GAE_DEPLOYMENT_ID") != ""
 }
 
-type gaeLogger struct {
+type sdLogger struct {
 	project string
 	monRes  *monitoredres.MonitoredResource
 	lc      *logging.Client
 	lgr     *logging.Logger
 }
 
-func newAppEngineLogger(ctx context.Context, projectID, service, version string) (log.Logger, func() error, error) {
+func newStackdriverLogger(ctx context.Context, logID, projectID, service, version string) (log.Logger, func() error, error) {
+	resource := &monitoredres.MonitoredResource{
+		Labels: map[string]string{
+			"module_id":  service,
+			"project_id": projectID,
+			"version_id": version,
+		},
+	}
+	if lid := os.Getenv("SD_LOG_ID"); lid != "" {
+		logID = lid
```

Comment:
Do folks need to be able to override this? Would we just set the convention that it'd be "stdout" if not in GAE?

Reply:
we are, we are logging to 3 different namespaces.

Reply:
removed